doc/sphinx-guides/source/admin/integrations.rst

```diff
@@ -77,6 +77,13 @@ Whole Tale
 `import data from Dataverse
 <https://wholetale.readthedocs.io/en/stable/users_guide/manage.html>`_ via identifier (e.g., DOI, URI, etc) or through the External Tools integration.  For installation instructions, see the :doc:`external-tools` section or the `Integration <https://wholetale.readthedocs.io/en/stable/users_guide/integration.html#dataverse-external-tools>`_ section of the Whole Tale User Guide.
 
+Binder
+++++++
+
+Researchers can launch Jupyter Notebooks based on the DOI of a Dataverse dataset from https://mybinder.org
```

Comment:
Is dataverse *only* for Jupyter Notebooks? Binder works with any kind of repository, so long as it follows the [reproducible executable environment specification](https://repo2docker.readthedocs.io/en/latest/specification.html).

Reply:
@choldgraf sorry, no, absolutely any file can be uploaded to Dataverse. We get a lot of Excel spreadsheets but truly a wide variety of file types. You can get a sense of the file types in Harvard Dataverse by looking at https://dataverse.harvard.edu/dataverse/harvard?q*=&types=files

Reply:
cool - well I'm not sure what would be the right language, but worth noting that if somebody uploads, say, a Julia repository or an RStudio repository, it would also work with repo2docker / binder.

Reply:
@choldgraf oh! Well, we have a lot of RStudio users for sure.

@djbrooke as you review this pull request, please take note! ☝️ Binder does more than I thought!

Reply:
@pdurbin I added some new text here. The rest of the documentation looks good. If my suggestion looks good, I think we can move this to QA.